README.md

```diff
@@ -68,17 +68,17 @@ pip install virtualenv
 #### On Debian-based Linuxes
 
 ``` sh
-sudo apt install python-virtualenv python-pip
+sudo apt install python3-virtualenv python3-pip
 ./mach bootstrap
 ```
 
-If you are on Ubuntu 20.04 or higher you need to do the following to install the missing python2 bits:
+If you are on Ubuntu 20.04 or higher you need to do the following to install the missing python3 bits:
```

Comment:
Don't have it working yet.. but if it's supposed to work with python3 this and the whole git-pip section that follows can be removed.

Reply:
Removed.
Thanks for the help